Makefile

```diff
@@ -2,6 +2,12 @@ ifndef TAG
     TAG := master
 endif
 
+ifndef os_version
+OS_VERSION=centos:7.6.1810
+else
+OS_VERSION=$(os_version)
```

Comment:
Seems if the user followed your example in this PR this value would not contain the container name, so it could not build?

Reply:
Addressd，now it can be build，PTAL